infra/src/Pos/Infra/Util/JsonLog/Events.hs

```diff
@@ -1,8 +1,3 @@
-{-# LANGUAGE TypeFamilies #-}
-
-{-# OPTIONS_GHC -fno-warn-orphans #-}
-
--- | Some types for json logging.
 module Pos.Infra.Util.JsonLog.Events
        ( HasJsonLogConfig (..)
        , JLEvent(..)
```

Comment:
I *think* this only excludes `InvReqDataFlowLog`, went into the sinbin from [here](https://github.com/input-output-hk/cardano-sl/pull/3203/files#diff-b113e527cb20b56d52305d2ca8b5f868L396). It gets imported & re-exported [here](https://github.com/input-output-hk/cardano-sl/pull/3203/files#diff-b113e527cb20b56d52305d2ca8b5f868R28).

Reply:
Yes, indeed. You read correctly! :)

I actually do think moving `InvReqDataFlowLog` is a little savage, but it was required if we wanted to move this module.